src/Catalog/Dnx/DnxMaker.cs

```diff
@@ -114,6 +149,7 @@ public async Task DeletePackageAsync(string id, string version, CancellationToke
 
             await DeleteNuspecAsync(storage, id, normalizedVersion, cancellationToken);
             await DeleteNupkgAsync(storage, id, normalizedVersion, cancellationToken);
+            await DeleteIconAsync(storage, id, normalizedVersion, cancellationToken);
```

Comment:
Is license deleted?

Reply:
Catalog2dnx does not extract the license, so it does not clean it up. It is deleted by Gallery though.

Reply:
Okay, out of scope then. Could you link the issue here tracking this improper design?

Reply:
https://github.com/NuGet/NuGetGallery/issues/7110